Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import java.util.Scanner;\r\n\r\npublic class Main {\r\n    public static void main(String[] args) {\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
--- a/src/Main.java	(revision 52006110ea0ef59dcc95968c4cb348593f85edf4)
+++ b/src/Main.java	(date 1742831429320)
@@ -2,5 +2,9 @@
 
 public class Main {
     public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        PizzaMenu menu = new PizzaMenu();
+        menu.addPizzaMenu(menu);
+        menu.changePrice(scanner);
     }
 }
\ No newline at end of file
